Tests/Observable+MoyaSpec.swift

```diff
@@ -326,6 +326,23 @@ class ObservableMoyaSpec: QuickSpec {
                 expect(receivedObject?.createdAt) == formatter.date(from: "1995-01-14T12:34:56")!
             }
 
+            it("maps data representing a json array at a key path to a decodable object") {
```

Comment:
nit: it would be nice to mark tests we add from bug reports with the bug report that found it. So adding something like bug from Moya/Moya#1311 would be awesome

Reply:
That's a great idea.